insights/specs/default.py

```diff
@@ -43,6 +43,21 @@ def get_owner(filename):
     return (name, group)
 
 
+def get_cmd_and_pacakge_in_ps(broker, target_command):
+        ps = broker[DefaultSpecs.ps_auxww].content
+        ctx = broker[HostContext]
+        results = set()
+        for p in ps:
+            p_splits = p.split(None, 10)
+            cmd = p_splits[10].split()[0] if len(p_splits) == 11 else ''
+            which = ctx.shell_out("which {0}".format(cmd)) if target_command in os.path.basename(cmd) else None
+            resolved = ctx.shell_out("readlink -e {0}".format(which[0])) if which else None
+            pkg = ctx.shell_out("rpm -qf {0}".format(resolved[0])) if resolved else None
+            if cmd and pkg:
```

Comment:
See [PEP8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) regarding recommendations on tests for `None`.